### ISSUE TITLE
Not support tsv files

### ISSUE THREAD

**Issue by ghost** [boakley/robotframework-workbench#9]

**Comment by boakley** [comment]
Correct, the editor currently only supports pipe-separated files. The other tools suport all formats. If there is enough demand I can add support for other formats to the editor. 

I will leave this issue open, though I don't plan on fixing this feature until I get more feedback.
